handler/src/test/java/io/netty/handler/ssl/BouncyCastlePemReaderTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2025 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.ssl;
+
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Isolated;
+
+import java.security.Provider;
+import java.security.Security;
+
+@Isolated("Provider value stored in the BouncyCastlePemReader#bcProvider field is also used by other tests")
+public class BouncyCastlePemReaderTest {
+
+    @Test
+    public void testBouncyCastleProviderLoaded() {
+        // tests org.bouncycastle.jce.provider.BouncyCastleProvider is detected as available
+        // because provider with matching name is present in 'java.security.Security'
+
+        Assertions.assertTrue(BouncyCastlePemReader.isAvailable());
+        Provider bcProvider = BouncyCastlePemReader.resetBcProvider();
+        Assertions.assertNotNull(bcProvider);
```

Comment:
nit: import static method.

Reply:
I did import static methods for `Assertions`, done, didn't do it for `Security` or `BouncyCastlePemReader` because there it is so obvious where the static method comes from (belongs) unless I look at the static imports. I can do it for these too, it is just my personal preference.

Reply:
But I can import every single static method if you feel strongly about it, just say a word.